utilfw/utilfw.go

```diff
@@ -0,0 +1,36 @@
+package utilfw
```

Comment:
Not sure we need another directory for this. Why not put this in `/util/framework.go`? Of if you like short name, `util/fw.go`?

Reply:
We need another package name. Can't have more than one package in the folder, if it's not `<packagename>_test`.

Reply:
Changed to `/util/sdk`  and `/util/fw`.